Project_source/app/src/main/java/android/clicker/school_live_simulator/ComputerCourseState.kt

```diff
@@ -0,0 +1,30 @@
+package android.clicker.school_live_simulator
+
+abstract class ComputerCourseState: StudyCourseState {
+    protected abstract val price: Int
+
+    /**
+     * This is a list of available web tasks
+     * These types are described in enumeration class WebTask
+     */
+
+    protected val available_web_task: List<WebTask> = listOf<WebTask>()
```

Comment:
@Ferrociraptus, what should we use - list or arraylist?

Reply:
@yevGen-S Use here arrayList instead of List pls

Reply:
Fixed